src/libopensc/pkcs15-jpki.c

```diff
@@ -139,7 +139,6 @@ sc_pkcs15emu_jpki_init(sc_pkcs15_card_t * p15card)
 		static int prkey_pin[2] = { 1, 2 };
 		static int prkey_usage[2] = {
 			SC_PKCS15_PRKEY_USAGE_ENCRYPT |
-			SC_PKCS15_PRKEY_USAGE_DECRYPT |
 			SC_PKCS15_PRKEY_USAGE_SIGN,
 			SC_PKCS15_PRKEY_USAGE_NONREPUDIATION
```

Comment:
Usually _non-repudiation_ usage has to be combined with _sign_ usage. It's a common practice for the "Digital Signature Key". 

The pure "Authentication" key has to be with _sign_ usage. 

'Encrypt' usage is usually used with the public key,
private key is usually used with 'Decrypt' flag.

Looking onto the context of _prkey_usage[2]_ data, I suggest that it's value has to be:
``` 
static int prkey_usage[2] = {
    SC_PKCS15_PRKEY_USAGE_SIGN,
    SC_PKCS15_PRKEY_USAGE_SIGN  | SC_PKCS15_PRKEY_USAGE_NONREPUDIATION
};
```

Please, confirm or object this suggestion and we will close you PR.